### DISCUSSION TITLE
Forced to do this, forced to do that, why can't it be simple?

### DISCUSSION THREAD

**Discussion by jpelizza** [community/community#78999] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I'm very sad about the state of github.
I was forced to move from a simple password system to a sha256 key.
Now I'm being force to have a authentication app on my phone at all times else I lose access to being able to work.

Now I'm forced to buy a second phone to be able to work and also don't have spyware on my pocket. I'll never understand how taking away choice from the user is a good thing, why not make it optional?

Is it for my safety? why am I not able to make a choice on it?

**Comment by erickdc7** [comment]
I understand your frustration with the recent changes in GitHub's authentication methods. The transition from a simple password system to a SHA-256 key, and the requirement to have an authentication app on your phone, can indeed be challenging. 

These changes are aimed at enhancing the security of user accounts. While they may seem inconvenient, they are designed to protect your data from potential threats. However, I understand that these changes may not suit everyone's needs or preferences.

Your feedback is valuable, and I hope GitHub takes your concerns into consideration. They are constantly working on improving the user experience, and user input is crucial in this process.
